get_positive_numbers.py

```diff
@@ -0,0 +1,8 @@
+def get_positive_numbers() -> list:
+    """
+        The function receives from the user a series of numbers separated by a comma.
+        It returns all the positive numbers that the user entered, as a list of integers.
+        :return: A list of integers.
+    """
+    return list(filter(lambda number: number > 0,
+                       map(int, input("Please enter a series of numbers separated by a comma: ").split(','))))
```

Comment:
In this case the assignment wasn't clear, but in general we prefer to seperate between IO and method logic.
https://wiki.c2.com/?SeparateIoFromCalculation

Reply:
Ok, thank you.